_posts/en/newsletters/2019-10-23-newsletter.md

```diff
@@ -0,0 +1,130 @@
+---
+title: 'Bitcoin Optech Newsletter #69'
+permalink: /en/newsletters/2019/10/23/
+name: 2019-10-23-newsletter
+slug: 2019-10-23-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces the release of C-Lightning 0.7.3,
+requests help testing a Bitcoin Core release candidate, invites
+participation in a taproot review club, highlights updates to two
+Bitcoin wallets, and describes a few notable changes to popular
+Bitcoin infrastructure projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Upgrade C-Lightning to version 0.7.3:** FIXME: mentioned in
+  6b1b99d7deb2d7a41fb93425239cc02854647427 but not tagged or published
+  yet
+
+- **Help test Bitcoin Core 0.19.0rc1:** production users of Bitcoin Core
+  are especially encouraged to test this [latest release
+  candidate][bitcoin core 0.19.0].
+
+## News
+
+- **Taproot review club:** starting the first week of November, several Bitcoin
+  contributors will be hosting a series of weekly meetings to help guide
+  people through review of the proposed [bip-schnorr][],
+  [bip-taproot][], and [bip-tapscript][] changes.  All developers,
+  academics, and anyone else with technical experience are welcome.  The
+  minimum expected commitment is four hours a week for seven weeks, with
+  one hour each week being a group meeting and the other three hours
+  being your own independent review of the proposals.  In addition to
+  review, developers will be encouraged to optionally implement a
+  proof-of-concept that either roughly integrates schnorr or taproot
```

Comment:
I don't think 'roughly' is needed here. You've already said it's a proof-of-concept.